consensus/db/db.go

```diff
@@ -1699,7 +1700,7 @@ func (pni *PendingLeafIter) Close() {
 
 func (db *Database) SetSafeToProceed(txn *badger.Txn, height uint32, isSafe bool) error {
 	if height%constants.EpochLength != 1 {
-		panic("The height must be mod 1 epoch length")
+		panic(fmt.Errorf("the height must be mod 1 epoch length. height: %v\n", height))
```

Comment:
nit: panic internally just converts the error to a string, which negates the need to do `fmt.Errorf`. You can just do a `fmt.Sprintf` if you want to convert to format a string. Also the trailing `\n` isn't needed as panic inserts one automatically.